impala/_rpc/hiveserver2.py

```diff
@@ -226,10 +224,46 @@ def get_result_schema(service, operation_handle):
         else:
             schema.append((name, type_, None, None, None, None, None))
 
-
     return schema
 
 
+_re_nulls = re.compile(b'^(\x00)+$')
+
+
+if six.PY3:
+
+    def _get_null(values, nulls, i):
+        if not isinstance(nulls, bytes):
+            nulls = nulls.encode('utf8')
```

Comment:
BTW this is total craziness. in some cases `nulls` is bytes and in others unicode. Must be a bug in thriftpy